ballerina/tests/tests.bal

```diff
@@ -13,18 +13,25 @@
 // KIND, either express or implied.  See the License for the
 // specific language governing permissions and limitations
 // under the License.
+
+import twilio.mock as _;
+
 import ballerina/http;
+import ballerina/log;
 import ballerina/os;
 import ballerina/test;
 
-configurable string accountSid = os:getEnv("ACCOUNT_SID");
-configurable string authToken = os:getEnv("AUTH_TOKEN");
-configurable string toPhoneNumber = os:getEnv("TO_PHONE");
-configurable string fromPhoneNumber = os:getEnv("TWILIO_PHONE");
+configurable boolean isTestOnLiveServer = os:getEnv("IS_TEST_ON_LIVE_SERVER") == "true";
+
+// Configurables
+configurable string accountSid = isTestOnLiveServer ? os:getEnv("ACCOUNT_SID"): "AC12345678901234567890123456789012";
```

Comment:
You have set the default values publically. Are these values dummy?

Reply:
Yes they are dummy data that expected from the mock server.